xbmc/ApplicationMessenger.cpp

```diff
@@ -1412,3 +1419,10 @@ void CApplicationMessenger::CECStandby()
   ThreadMessage tMsg = {TMSG_CECSTANDBY};
   SendMessage(tMsg, false);
 }
+
+void CApplicationMessenger::SetPVRManager(bool onOff)
```

Comment:
Thanks @xhaggi for review, I rename it want to have amount of functions and related message id's so low as possible